localization/ekf_localizer/include/ekf_localizer/warning_message.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Autoware Foundation
```

Comment:
I will approve after the conflict is resolved!

Reply:
Resolved the conflicts